libubx/Makefile.am

```diff
@@ -24,7 +24,7 @@ BUILT_SOURCES = ubx_proto.h
 CLEANFILES = $(BUILT_SOURCES)
 
 ubx_proto.h: md5.c ubx.c
-	cproto $(srcdir)/md5.c $(srcdir)/ubx.c -I$(top_builddir)/ -I${top_srcdir}/ -I${top_srcdir}/libubx/ > $(srcdir)/$@
+	/usr/bin/cproto $(srcdir)/md5.c $(srcdir)/ubx.c -I$(top_builddir)/ -I${top_srcdir}/ -I${top_srcdir}/libubx/ > $(srcdir)/$@
```

Comment:
Why do we need to add a fixed path here?

Reply:
I looked for a cproto recipe in Yocto a few weeks ago and came up with nothing. Also, the ubx_proto.h file is checked into the git repo, and it should surely not be there and should be in the .gitgnore? However, if any of the files used to generate cproto are touched after the checkout, the build system attempts to re-generate ubx_proto.h, and as there is no cproto in the Yocto environment, it fails saying it cannot find cproto. I guess I could go and create a Yocto recipe to build cproto for the build environment, but I thought as we are building for x86 anyway and we have cproto installed on the build machine, I would hack it, being lazy again!